Xamarin.PropertyEditing.Mac/Controls/Custom/CommandMenuItem.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Windows.Input;
+using AppKit;
+
+namespace Xamarin.PropertyEditing.Mac
+{
+	public class CommandMenuItem : NSMenuItem
+	{
+		ICommand command;
+
+		public ICommand Command {
+			get { return command; }
+			set {
+				if (this.command != null)
+					this.command.CanExecuteChanged -= CanExecuteChanged;
+
+				this.command = value;
+				this.command.CanExecuteChanged += CanExecuteChanged;
+			}
+		}
+
+		public CommandMenuItem (string title) : base (title)
```

Comment:
Can you move constructors up?

Reply:
Sorry, I'm not sure what you mean here?

Reply:
I mean above the Command property.

Reply:
Oh, I always tend to have variables/properties etc above constructors, methods and the like. Not always but mostly. Is having the constructors at the top part of our coding standards? If so I've missed that the last 5 years, or is it a MS specific thing?

Reply:
Not coding standards as far as I know, but usage in the rest of the codebase, certainly.

Reply:
@bleroy ^ in case you missed my reply.